inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -47,11 +47,9 @@
                 <div class="brief-record-details">
                 <div class="authors">
                   {{ render_record_authors(is_brief=true, show_affiliations=false) }}
-                  {% if record.get('earliest_date') %}
-                    <span id="record-date">
-                      - {{ record.get('earliest_date').split('-')[0] }}
-                    </span>
-                  {% endif %}
+                  <span id="record-date">
+                    - {{ record.get('earliest_date')|format_date() }}
```

Comment:
Don't know why it gave me the impression to not work properly yesterday without calling it. But just tried again and seems fine. I will amend it.

So we can standarize:

No parameters
```python
{{ record.get('earliest_date')|format_date }}
```

Parameters
```python
{{ record.get('earliest_date')|format_date(extra_params) }}
```